### DISCUSSION TITLE
Unable to run XE images on Windows 11 due to permission issues

### DISCUSSION THREAD

**Discussion by franco-caponi-ws** [gvenzl/oci-oracle-xe#228] [Q&A]
Hi all,
i'm facing issues trying to use these wonderful images on my pc running windows 11 (on docker desktop set as linux), while they work without issues on a network pc based on linux.

The images run without issues with these commands:
mkdir /home/franco/.oracle_data
chown 1000:1000 /home/franco/.oracle_data
docker run -d \
	-p 1521:1521 \
	-e ORACLE_PASSWORD=website \
	-v /home/franco/.oracle_data:/opt/oracle/oradata/XE \
	gvenzl/oracle-xe:11.2.0.2-full-faststart

but when I issue create tablespace command 
CREATE TABLESPACE MY_TABLESPACE
   DATAFILE '/opt/oracle/oradata/XE/MY_TABLESPACE.DBF' 
   SIZE 5M AUTOEXTEND ON NEXT 5M MAXSIZE UNLIMITED;
i got permission issue
Linux-x86_64 Error: 13: Permission denied

Any hint?

**Comment by pranithan-kang** [comment]
Maybe the directory in the host (aka. Windows) `C:\Users\your_name\path-to\oracle_data` do not permit the docker system to write?
